xml/vt_docker_quick.xml

```diff
@@ -328,7 +328,7 @@
    </para>
 <!--taroth 2015-07-02: http://doccomments.provo.novell.com/comments/28724 -->
    <procedure>
-    <title>Enabling the Container Module</title>
+    <title>Enabling the Container Module - interactive way</title>
```

Comment:
What about `Enabling the Controller Module Using &yast;` and `Enabling the Controller Module Using SUSEConnect` ?